pkg/asset/installconfig/azure/session.go

```diff
@@ -16,15 +16,15 @@ import (
 
 const azureAuthEnv = "AZURE_AUTH_LOCATION"
 
-var authFilePath = os.Getenv("HOME") + "/.azure/osServicePrincipal.json"
+var defaultAuthFilePath = os.Getenv("HOME") + "/.azure/osServicePrincipal.json"
```

Comment:
I prefer `filePath.Join` so we aren't so far off when we get around to compiling for Windows.  But whatever, the `+` concat approach is what we had before this PR.

Reply:
fixed with https://github.com/openshift/installer/compare/4bff9ab1f5f879ccdabd9fa5730d21b711aec476...0e9453e723a95ec3a6290e2985d7926f8a3f7618